src/com/twu/biblioteca/BibliotecaApp.java

```diff
@@ -6,7 +6,7 @@
     public static void main(String[] args) {
 
         MainMenu mainMenu = new MainMenu();
-        mainMenu.welcome();
+        mainMenu.showWelcomeMessage();
```

Comment:
Good!  This is more consistent and aligned with the naming of the other MainMenu methods.